service/controller/chart/v1/resource/release/create.go

```diff
@@ -52,15 +52,18 @@ func (r *Resource) ApplyCreateChange(ctx context.Context, obj, createChange inte
 		err = r.helmClient.InstallReleaseFromTarball(ctx, tarballPath, ns, helm.ReleaseName(releaseState.Name), helm.ValueOverrides(releaseState.ValuesYAML))
 		if err != nil {
 			releaseContent, err := r.helmClient.GetReleaseContent(ctx, releaseState.Name)
-			if err != nil {
+			if helmclient.IsReleaseNotFound(err) {
+				r.logger.LogCtx(ctx, "level", "debug", "message", fmt.Sprintf("helm release not found %#q", releaseContent.Name))
+				r.logger.LogCtx(ctx, "level", "debug", "message", "canceling resource")
+				resourcecanceledcontext.SetCanceled(ctx)
+				return nil
```

Comment:
If there is no helm release we should cancel the resource rather than triggering the operatorkit errors alert.

We have alerting for apps not being installed and it is then clearer what the problem is.